lib/features/song_detail/presentations/song_details_screen.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:palakat/core/assets/assets.dart';
+import 'package:palakat/core/constants/constants.dart';
+import 'package:palakat/core/utils/extensions/extension.dart';
+import 'package:palakat/core/widgets/scaffold/scaffold_widget.dart';
+import 'package:palakat/core/widgets/screen_title/screen_title_widget.dart';
+import 'package:palakat/features/song_detail/presentations/song_details_data.dart';
+
+class SongDetails extends StatelessWidget {
+  SongDetails({super.key});
+
+  final DummyData dummyData = DummyData();
+
+  @override
+  Widget build(BuildContext context) {
+    return ScaffoldWidget(
+      child: Column(
+        children: [
+          ScreenTitleWidget.primary(
+            title: dummyData.titles,
+            subTitle: dummyData.subtitles,
+            leadIcon: Assets.icons.line.chevronBackOutline,
+            leadIconColor: BaseColor.black,
+            onPressedLeadIcon: () {
+              Navigator.pop(context);
+            },
+          ),
+          Gap.h24,
+          Align(
```

Comment:
Nah skarang coba cek di dalam widget Text, ada properti for textAlignment kalo nda salah, jadi nda perlu si widget text mo bungkus dalam widget Align

Reply:
klo cmn pake textAlign nda mo ta ka kiri dpe verse itu

Reply:
jadi mse bagimana itu

Reply:
![image](https://github.com/user-attachments/assets/011064c6-9366-4479-b13b-5a0299c3aefe)

Reply:
d p column tambah crossAxisAlignment beking stretch